community/server/pom.xml

```diff
@@ -388,8 +388,8 @@
           <classpathScope>test</classpathScope>
           <systemProperties>
             <systemProperty>
-              <key>org.neo4j.server.properties</key>
-              <value>${basedir}/neo4j-home/conf/neo4j-server.properties</value>
+              <key>org.neo4j.config.file</key>
+              <value>${basedir}/neo4j-home/conf/neo4j.properties</value>
```

Comment:
Entirely outside this PR: Probably worth reviewing if we still need that neo4j-home folder and the associated config. Ideally all the server tests should be confined to things that go under `target/` nowadays. The neo4j-home checked into the source tree is a very, very old annoyance.